tests/integration/helpers.bash

```diff
@@ -8,6 +8,9 @@ INTEGRATION_ROOT=$(dirname "$(readlink -f "$BASH_SOURCE")")
 RUNC="${INTEGRATION_ROOT}/../../runc"
 RECVTTY="${INTEGRATION_ROOT}/../../contrib/cmd/recvtty/recvtty"
 GOPATH="$(mktemp -d --tmpdir runc-integration-gopath.XXXXXX)"
+SECCOMPHOOKSOURCE="${INTEGRATION_ROOT}/../../contrib/cmd/seccomphook/seccomphook"
+SECCOMPHOOKDESTINATION="/tmp/seccomphook"
```

Comment:
Could we use a unique directory with mktemp?

Reply:
I updated the code to pass the full path of the hook to avoid copying the binary around.